### PR TITLE
Adding Nav tabs code page & changing tabs into nav tabs in website

### PR DESCRIPTION
**Pull request by Jialecl**

### CODE REVIEW

File: website/screens/components/nav-tabs/NavTabsPageLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}: {
+  children: React.ReactNode;
+}) => {
+  const tabs = [{ label: "Code", path: "/components/nav-tabs" }];
+
+  return (
+    <DxcStack gutter="xlarge">
+      <PageHeading>
+        <DxcStack gutter="large">
+          <DxcHeading level={1} text="Number Input" weight="bold"></DxcHeading>
```
Comment (Lisa159) [inline website/screens/components/nav-tabs/NavTabsPageLayout.tsx:16]: Change Number Input to Nav Tabs.

**Comment by Lisa159** [comment]
Props for child `Tab` are missing in the documentation.

File: website/screens/components/nav-tabs/NavTabsPageLayout.tsx (hunk 1 above)
Comment (Jialecl) [reply website/screens/components/nav-tabs/NavTabsPageLayout.tsx]: done

File: website/screens/components/nav-tabs/code/NavTabsCodePage.tsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+              </>
+            ),
+          },
+        ],
+      },
+    ],
+  },
+];
+
+const ButtonCodePage = () => {
```
Comment (Lisa159) [inline website/screens/components/nav-tabs/code/NavTabsCodePage.tsx:219]: Change name variable

Comment (Jialecl) [reply website/screens/components/nav-tabs/code/NavTabsCodePage.tsx]: done

**Pull request merged**